File: using-java/src/Q07_AlphabeticPattern.java
public class Q07_AlphabeticPattern {

    public static void main(String[] args) {
        int rows = 5;
        for (int i = 1; i <= rows; i++) {
            char ch = 'A';
            for (int j = 1; j <= i; j++) {
                System.out.print(ch++);
            }
            System.out.println();
        }
    }

    /*

    Alphabetic Pattern

    A
    AB
    ABC
    ABCD
    ABCDE



     */
}

File: using-python/Q07_AlphabeticPattern.py
def alphabetic_pattern(size):
    for i in range(size):
        output = ""
        for j in range(i + 1):
            output += chr(65 + j)
        print(output)


alphabetic_pattern(5)


# A
# AB
# ABC
# ABCD
# ABCDE
